README.md

```diff
@@ -166,15 +166,23 @@ apt install python3 libpq-dev python3-dev python3-venv
 Here's how to quickly get started:
 
 ```bash
+
+# Start up backing services (database and database admin)
+# NOTE: Remember to run "docker-compose down" afterwards
+docker-compose up --detach
```

Comment:
Mmhh.. this is a bit confusing. In the instruction I'm explaining how to run the application locally, why should we suggest to bring up the docker-compose setup and then provide instructions to set stuff up outside of the containers?

Reply:
Well, without `docker-compose up`, the `python manage.py migrate` command mentioned in the setup will fail with 

```
django.db.utils.OperationalError: connection to server at "0.0.0.0", port 5432 failed: Connection refused
Is the server running on that host and accepting TCP/IP connections?
```

since the `settings.py` specify a `"default"` PostgreSQL database on port 0.0.0.0.

In other words, the application setup won't work currently as-is without the database, and I figured using `docker compose` was a good way for contributors to get up and running quickly since it was already defined in the codebase

Reply:
@rnegron I tagged you on Discord, it may be easier to iterate there